app/scripts/components/common/form/checkable-filter/index.tsx

```diff
@@ -83,10 +84,18 @@ export interface OptionItem {
 }
 
 export default function CheckableFilters(props: CheckableFiltersProps) {
-  const {items, title, onChanges, globallySelected, tagItemCleared} = props;
-  const [show, setShow] = useState<boolean>(false);
+  const {
+    items,
+    title,
+    onChanges,
+    globallySelected,
+    tagItemCleared,
+    calculateHeightCallback
+  } = props;
+  const [show, setShow] = useState<boolean>(true);
   const [count, setCount] = useState<number>(0);
   const [selected, setSelected] = useState<OptionItem[]>([]);
+  const targetRef = React.useRef<HTMLDivElement>(null);
```

Comment:
Is there any need to introduce the pattern of using `React.useRef`? I am not sure we can set it up as a lint rule, but all other parts of the code explicitly import ` {useRef} `

Reply:
not a pattern, can just explicitly import or use like this